.gitignore

```diff
@@ -22,3 +22,4 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+*.iml
```

Comment:
Why this commit is in this PR?

Reply:
It was my mistake. As a base branch, I used some feature branch instead of develop. 
I removed that commit from this branch.